Sources/_CryptoExtras/AES/AES_CFB.swift

```diff
@@ -0,0 +1,71 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCrypto open source project
+//
+// Copyright (c) 2024 Apple Inc. and the SwiftCrypto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.md for the list of SwiftCrypto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Crypto
+import Foundation
+
+@usableFromInline
+typealias AESCFBImpl = OpenSSLAESCFBImpl
+
+extension AES {
+    public enum _CFB {
+        @inlinable
+        public static func encrypt<Plaintext: DataProtocol>(
+            _ plaintext: Plaintext,
+            using key: SymmetricKey,
+            nonce: AES._CFB.Nonce
+        ) throws -> Data {
+            let bytes: ContiguousBytes = plaintext.regions.count == 1 ? plaintext.regions.first! : Array(plaintext)
+            return try AESCFBImpl.encryptOrDecrypt(.encrypt, bytes, using: key, nonce: nonce)
+        }
+
+        @inlinable
+        public static func decrypt<Ciphertext: DataProtocol>(
+            _ ciphertext: Ciphertext,
+            using key: SymmetricKey,
+            nonce: AES._CFB.Nonce
+        ) throws -> Data {
+            let bytes: ContiguousBytes = ciphertext.regions.count == 1 ? ciphertext.regions.first! : Array(ciphertext)
+            return try AESCFBImpl.encryptOrDecrypt(.decrypt, bytes, using: key, nonce: nonce)
+        }
+    }
+}
+
+extension AES._CFB {
+    public struct Nonce: Sendable {
```

Comment:
FYI: Our existing AES APIs differ in calling this `IV` or `Nonce`. I aligned with CTR, which was added most recently(?).

Reply:
For CFB nonce is acceptable, though it is probably more commonly called an IV. NIST calls it an IV, though that seems more to be out of caution than anything else.

Reply:
Renamed `AES._CFB.Nonce` to `IV` and all `nonce` and `nonceBytes` parameters to `iv` and `ivBytes, respectively.